src/rgw/driver/rados/rgw_rados.h

```diff
@@ -450,7 +451,7 @@ class RGWRados
 public:
   RGWRados(): timer(NULL),
                gc(NULL), lc(NULL), obj_expirer(NULL), use_gc_thread(false), use_lc_thread(false), quota_threads(false),
-               run_sync_thread(false), run_reshard_thread(false), meta_notifier(NULL),
+               run_sync_thread(false), run_reshard_thread(false), run_notification_thread(false), meta_notifier(NULL),
```

Comment:
i'd like to get in the habit of using member initializers instead of constructors for this, even if it's not consistent with other member variables in the class

Reply:
> i'd like to get in the habit of using member initializers instead of constructors for this, 

agree

> even if it's not consistent with other member variables in the class

i have to disagree here, as it would unnecessarily raise a question by someone reading the code - "why is that member different than the other?"

Reply:
i'd just like to be able to make incremental progress. if you'd like to replace all of these with member initializers, that's great too but it does add unrelated changes to the diff

Reply:
see: b1c4b7c471cbf3eada502a74764ffc6aac71df5b